### DISCUSSION TITLE
Support for Z-Push?

### DISCUSSION THREAD

**Discussion by SinghNanak** [docker-mailserver/docker-mailserver#4078] [Q&A]
How do you guys support ActiveSync? I am very interested in z-push's capabilities.

**Comment by polarathene** [comment]
I don't think we have any official support, at least I'm not aware of any maintainers that use such, nor is it a common topic with users.

With LDAP users, I think some may have used ActiveSync, but it is not as well supported. No clue regarding z-push.

**Reply by SinghNanak** [reply]
I'm asking because [Z-Push](https://github.com/Z-Hub/Z-Push) is a opensource implementation of ActiveSync, and compared to other activesync implementations it supports ActiveSync policies.

**Reply by polarathene** [reply]
As stated, no maintainer would be able to assist much. Maybe another community member will chime in eventually.

I do have a branch I'm working on to revise the LDAP docs and a PR to improve our LDAP config support, so once those are ready and merged, it might make it a bit easier for you to integrate. For the most part the LDAP support is community contributed.

Z-Push seems to have support with mail-in-a-box, perhaps that's an alternative that would work for you?: https://github.com/Z-Hub/Z-Push/wiki/Installation#z-push-on-mail-in-a-box

**Reply by SinghNanak** [reply]
thank you polarathene for clarifying.
